docs/topics/gradle/gradle-compilation-and-caches.md

```diff
@@ -1,4 +1,4 @@
-[//]: # (title: Compilation and caches)
+[//]: # (title: Compilation and caches in the Kotlin Gradle plugin)
```

Comment:
@koshachy, I understood that we can't say "in Gradle" because we tell users about our Kotlin Gradle plugin.